frontend_tests/node_tests/general.js

```diff
@@ -64,6 +64,7 @@ zrequire("stream_data");
 run_test("stream_data", () => {
```

Comment:
I am working on this commit, opened to inform what work I think was left in #14638